new/asset-lineage/egeria-asset-lineage.container.js

```diff
@@ -276,6 +281,52 @@ class EgeriaAssetLineage extends mixinBehaviors([EgeriaItemUtilsBehavior, RoleCo
     }
   }
 
+  onLinkClick({ linkElement }) {
+
+    if (this.sameLink(linkElement, this.selectedLink)) {
+      this.invertLinkSelection(linkElement)
+    } else {
+      this.selectLink(linkElement)
+
+      if(this.selectedLink){
+        this.deselectLink(this.selectedLink)
+      }
+    }
+    this.selectedLink = linkElement
+  }
+
+  sameLink(link, otherLink) {
+    return link === otherLink
+  }
+
+  invertLinkSelection(linkElement){
+    if(this.isLinkSelected(linkElement)){
+      this.deselectLink(linkElement)
+    }else{
+      this.selectLink(linkElement)
+    }
+  }
+
+  isLinkSelected(linkElement){
+    let stroke = linkElement.style.getPropertyValue("stroke");
+    let stroke_width = linkElement.style.getPropertyValue("stroke-width")
+    let marker_end = linkElement.getAttribute("marker-end")
+
+    return stroke === "var(--happi-graph-primary-color)" && stroke_width === "4" && marker_end === "url(#arrow-end-selected)";
```

Comment:
Instead of value "4" introduce a style var(--happi-graph-selected-stroke-width). This will make life easier if the value needs to be adjusted